src/lib.rs

```diff
@@ -104,6 +104,8 @@ pub use workspace_folders::*;
 mod workspace_symbols;
 pub use workspace_symbols::*;
 
+pub mod lsif;
```

Comment:
Could you add a comment explaining `lsif`/linking to the definition for it here so it shows up in docs?